### PR TITLE
Helium now makes your speech small!

### PR DESCRIPTION
**Pull request by SnoopCooper**
## About The Pull Request
Breathing helium now makes your voice appear small in the chat.
helium_text

## Why It's Good For The Game
Adds a funny interaction to helium.

## Changelog

:cl:
expansion: Gives helium a minor funny interaction
/:cl:

### CODE REVIEW

**Comment by ShizCalev** [comment]
does #66156 affect this

File: code/modules/surgery/organs/lungs.dm [hunk 1]
```diff
@@ -381,6 +385,18 @@
...
+		//Activates helium speech when partial pressure gets high enough
+		var/helium_pp = breath.get_breath_partial_pressure(breath_gases[/datum/gas/helium][MOLES])
+		if(helium_pp > helium_speech_min && !helium_speech)
+			helium_speech = TRUE
+			RegisterSignal(owner, COMSIG_MOB_SAY, .proc/handle_helium_speech)
+		else if(helium_pp <= helium_speech_min && helium_speech)
+			helium_speech = FALSE
+			UnregisterSignal(owner, COMSIG_MOB_SAY)
+		gas_breathed = breath_gases[/datum/gas/helium][MOLES]
+		breath_gases[/datum/gas/helium][MOLES]-=gas_breathed
```
Comment (Kylerace) [inline code/modules/surgery/organs/lungs.dm:398]:
```suggestion
		breath_gases[/datum/gas/helium][MOLES] -= gas_breathed
```

**Review by Kylerace — CHANGES_REQUESTED** [review]

**Comment by Ghilker** [comment]
Great pr, but you added the Bz reaction rework in this one

**Comment by SnoopCooper** [comment]
> Great pr, but you added the Bz reaction rework in this one

I know, trying to fix

**Comment by SnoopCooper** [comment]
I just made a big mess. Gonna redo this PR.

**Pull request closed without merging**